client/my-sites/plans-features-main/index.jsx

```diff
@@ -351,16 +351,15 @@ PlansFeaturesMain.PropTypes = {
 	hideFreePlan: PropTypes.bool,
 	showFAQ: PropTypes.bool,
 	selectedFeature: PropTypes.string,
-	displayJetpackPlans: PropTypes.bool
+	displayJetpackPlans: PropTypes.bool.isRequired
 };
 
 PlansFeaturesMain.defaultProps = {
 	basePlansPath: null,
 	intervalType: 'yearly',
 	hideFreePlan: false,
 	site: {},
-	showFAQ: true,
-	displayJetpackPlans: false
```

Comment:
Removing this default as `PlansFeaturesMain` is a reusable component which is used on multiple places (such as wpcom and Jetpack NUX).